fc/templates/weblab_model.pyx

```diff
@@ -312,5 +299,5 @@ cdef class {{ class_name }}(CvodeSolver):
         """
         # TODO Update this (and rest of fc) to Python3
         # TODO Use logging here, or raise an exception
-        print >>sys.stderr, '  ' * self.indent_level, 'set_solver: Models implemented using Cython contain a built-in ODE solver, so ignoring setting.'
+        print >>sys.stderr, 'set_solver: Models implemented using Cython contain a built-in ODE solver, so ignoring setting.'
```

Comment:
Is there a reason for not using logging here?

Reply:
Just that I'm not sure how to do it in cython